tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "compilerOptions": {
     "module": "commonjs",
     "target": "es5",
+    "lib": ["es6", "dom"],
```

Comment:
could you please point what does it solve?

Reply:
If I run `npm run build` I get the following errors:

```
node_modules/io-ts-types/lib/fp-ts/createSetFromArray.d.ts:9:163 - error TS2304: Cannot find name 'Set'.

9 export declare const createSetFromArray: <RT extends t.Type<A, O, t.mixed>, A = any, O = A>(type: RT, ordA: Ord<RT["_A"]>, name?: string) => SetFromArrayType<RT, Set<RT["_A"]>, RT["_O"][], t.mixed>;
```

This can be solved by adding `es6`. Now I get another error.

```
src/__tests__/remote-data.spec.ts:478:18 - error TS2304: Cannot find name 'ProgressEvent'.

478             const e = new ProgressEvent('test');
                              ~~~~~~~~~~~~~


src/remote-data.ts:916:48 - error TS2304: Cannot find name 'ProgressEvent'.

916 export function fromProgressEvent<L, A>(event: ProgressEvent): RemoteData<L, A> {
                                                   ~~~~~~~~~~~~~


src/remote-data.ts:916:48 - error TS4078: Parameter 'event' of exported function has or is using private name 'ProgressEvent'.

916 export function fromProgressEvent<L, A>(event: ProgressEvent): RemoteData<L, A> {
```

which I can solve by adding `dom`.